Python 3.7.5 (tags/v3.7.5:5c02a39a0b, Oct 15 2019, 00:11:34) [MSC v.1916 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license()" for more information.
>>> 2 + 3
5
>>> 9 - 8
1
>>> 4 * 6
24
>>> 8 / 4
2.0
>>> 5/2
2.5
>>> 5 //2
2
>>> 8+9-10
7
>>> 8+9-
SyntaxError: invalid syntax
>>> 8+2*3
14
>>> (8+2)*3
30
>>> 2*2*2
8
>>> 2**3
8
>>> 10//3
3
>>> 10 %3
1
>>> 'navin'
'navin'
>>> print("navin")
navin
>>> print('sandhya's laptop')
      
SyntaxError: invalid syntax
>>> print("sandhya's laptop")
sandhya's laptop
>>> print('sandhya "laptop"')
sandhya "laptop"
>>> 
>>> 
>>> 



>>> 




>>> 

>>> 

>>> print('navin's "laptop"')
      
SyntaxError: invalid syntax
>>> print('navin\'s "laptop"')
navin's "laptop"
>>> 'navin' * 3
'navinnavinnavin'
>>> 'navin\n' * 3
'navin\nnavin\nnavin\n'
>>> 'navin'\n * 3
SyntaxError: unexpected character after line continuation character
>>> 'navin' + 'navin'
'navinnavin'
>>> 10* 'navin'
'navinnavinnavinnavinnavinnavinnavinnavinnavinnavin'
>>> print('c:\docos\navin')
c:\docos
avin
>>> 'navin \n' * 3
'navin \nnavin \nnavin \n'
>>> print('c:\docos\navin')
c:\docos
avin
>>> print(r'c:\docos\navin')
c:\docos\navin
>>> x=2
>>> x+3
5
>>> y=3
>>> x+y
5
>>> x=9
>>> x+y
12
>>> x
9
>>> abc
Traceback (most recent call last):
  File "<pyshell#43>", line 1, in <module>
    abc
NameError: name 'abc' is not defined
>>> x+10
19
>>> 19+y
22
>>> _+y
25
>>> __+x
Traceback (most recent call last):
  File "<pyshell#47>", line 1, in <module>
    __+x
NameError: name '__' is not defined
>>> name="youtube"
>>> name
'youtube'
>>> name + 'rocks'
'youtuberocks'
>>> name + ' rocks'
'youtube rocks'
>>> name 'rocks'
SyntaxError: invalid syntax
>>> name[0]
'y'
>>> name[6]
'e'
>>> name[8]
Traceback (most recent call last):
  File "<pyshell#55>", line 1, in <module>
    name[8]
IndexError: string index out of range
>>> name[-1]
'e'
>>> name[-2]
'b'
>>> name[-7]
'y'
>>> name[0:2]
'yo'
>>> name[1:4]
'out'
>>> name[1:]
'outube'
>>> name[:4]
'yout'
>>> name[3:10]
'tube'
>>> name[0:3]='my'
Traceback (most recent call last):
  File "<pyshell#64>", line 1, in <module>
    name[0:3]='my'
TypeError: 'str' object does not support item assignment
>>> name[0]
'y'
>>> name[0]='r'
Traceback (most recent call last):
  File "<pyshell#66>", line 1, in <module>
    name[0]='r'
TypeError: 'str' object does not support item assignment
>>> 'my' =name[3]
SyntaxError: can't assign to literal
>>> 'my' +name[3:]
'mytube'
>>> my='Sandhya Rani'
>>> len(my)
12
>>> print(r,'Telusko \n Rocks')
Traceback (most recent call last):
  File "<pyshell#71>", line 1, in <module>
    print(r,'Telusko \n Rocks')
NameError: name 'r' is not defined
>>> print(r'Telusko\n Rocks')
Telusko\n Rocks
>>> print(r'Telusko \n Rocks')
Telusko \n Rocks
>>> nums=[25,12,95,14,36]
>>> nums
[25, 12, 95, 14, 36]
>>> num[0]
Traceback (most recent call last):
  File "<pyshell#76>", line 1, in <module>
    num[0]
NameError: name 'num' is not defined
>>> nums[0]
25
>>> nums[4]
36
>>> nums[2:]
[95, 14, 36]
>>> nums[-1]
36
>>> nums[5]
Traceback (most recent call last):
  File "<pyshell#81>", line 1, in <module>
    nums[5]
IndexError: list index out of range
>>> nums[-5]
25
>>> names=['navin','kiran','john']
>>> names
['navin', 'kiran', 'john']
>>> values=[9.5,'navin',25]
>>> values
[9.5, 'navin', 25]
>>> mil=[nums,names]


          
